Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Arpita S.K <arpita.s.k@arm.com>

From dbaf7a11e686d362eb09e63841eb718ea777dd03 Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Wed, 1 Sep 2021 16:46:42 +0100
Subject: [PATCH] plat-corstone1000: reserve 3MB CVM memory for optee

optee requires 3MB CVM memory to include SecurePartition's into
it's image

Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>

diff --git a/core/arch/arm/plat-corstone1000/conf.mk b/core/arch/arm/plat-corstone1000/conf.mk
index b14dd442..7a4aa13c 100644
--- a/core/arch/arm/plat-corstone1000/conf.mk
+++ b/core/arch/arm/plat-corstone1000/conf.mk
@@ -23,8 +23,7 @@ CFG_WITH_LPAE ?=y
 
 CFG_TEE_CORE_NB_CORE = 1
 CFG_TZDRAM_START ?= 0x02002000
-CFG_TZDRAM_SIZE  ?= 0x000FE000
-CFG_TEE_RAM_VA_SIZE ?= 0x00AF000
+CFG_TZDRAM_SIZE  ?= 0x300000 # OPTEE CODE + DATA +TA_RAM = 3MB
 CFG_SHMEM_START  ?= 0x86000000
 CFG_SHMEM_SIZE   ?= 0x00200000
 
-- 
2.17.1

